# Default initial positions for tm5-900's ros2_control fake system

initial_positions:
  joint_1: 0
  joint_2: 0
  joint_3: 0
  joint_4: 0
  joint_5: 0
  joint_6: 0
  robotiq_85_left_knuckle_joint: 0
  robotiq_85_right_knuckle_joint: 0